frontend/src/metabase/actions/components/ActionViz/ActionVizForm.tsx

```diff
@@ -140,7 +140,7 @@ function ActionVizForm({
             action={action}
             mappedParameters={mappedParameters}
             initialValues={initialValues}
-            prefetchesInitialValues
+            prefetchesInitialValues={shouldPrefetch}
```

Comment:
This part of the code is from https://github.com/metabase/metabase/pull/32671 (which comes after #32433), which is a bit weird because `git bisect` pointed out that https://github.com/metabase/metabase/pull/32433 caused it.

But in https://github.com/metabase/metabase/pull/32671, you could see that we [missed passing `shouldPrefetch` to `prefetchesInitialValues`](https://github.com/metabase/metabase/pull/32671/files#diff-00ad554d2261a533d6f01fbc893c58e0099c9d9ce29ca7d1242d8631facac1bcR143), which caused the bug. That's why I was confused about how https://github.com/metabase/metabase/pull/32433 introduced the bug in the first place.